src/update/UpdateManager.cpp

```diff
@@ -247,6 +248,10 @@ void UpdateManager::GetUpdates(bool ignoreNetworkErrors)
 #if UPDATE_LOCAL_TESTING == 0
         else // mVersionPatch.version > CurrentBuildVersion()
         {
+            gAudioIO->CallAfterRecording([] {
+                NoUpdatesAvailableDialog(nullptr).ShowModal();
```

Comment:
This dialog contains a suggestion to turn on update checking in preferences.  But maybe that suggestion should be conditional on `configurableNotification == false` ?  Should it be part of the dialog only when you use the Help menu, and not if it opens automatically?